.github/workflows/build-and-test.yml

```diff
@@ -695,6 +695,11 @@ jobs:
           mv convert_graphdef_memmapped_format /tmp
       - run: |
           mkdir -p ${CI_ARTIFACTS_DIR} || true
+      - name: Run python notebooks
+        run: |
+          sudo apt-get install -y --no-install-recommends libopusfile0 libopus-dev libopusfile-dev
+          python -m pip install jupyter
+          ./ci_scripts/notebook-tests.sh
```

Comment:
Don't add this to the basic training tests, it'll slow them down and they have downstream jobs that depend on them. Add a separate job for the notebook tests.

Reply:
Addressed in c78f98a